settings.py

```diff
@@ -54,7 +55,20 @@ def update(setting_name, updated_value):
     update_settings_file(settings)
 
 def setup_logger():
+    handlers = []
+    fh = logging.FileHandler(LOG_FILE_PATH)
+    fh.setLevel(logging.INFO)
+    handlers.append(fh)
+    if LOGGING_DEBUG:
+        sh = logging.StreamHandler()
```

Comment:
https://docs.python.org/3/library/logging.handlers.html#streamhandler

By default it writes to stderr, maybe it's better to set stdout